app/models/http_proxy.rb

```diff
@@ -0,0 +1,26 @@
+class HttpProxy < ActiveRecord::Base
+  include Authorizable
+  include Taxonomix
+  include Encryptable
+
+  extend FriendlyId
+
+  encrypts :password
+  friendly_id :name
+
+  has_many :compute_resources
+
+  validates :url, :presence => true
```

Comment:
This should also validate, that a user entered a correct url with schema.
`http://example.com` should be allowed, `börks` should be invalid.